ppr-ui/src/components/unitNotes/UnitNotePanels.vue

```diff
@@ -0,0 +1,348 @@
+                  fixed-header
+                >
+                  <template v-slot:default>
+                    <!-- Table Headers -->
+                    <thead>
+                      <tr>
+                        <th v-for="header in personGivingNoticeTableHeaders" :key="header.value" :class="header.class">
+                          {{ header.text }}
+                        </th>
+                      </tr>
+                    </thead>
+
+                    <!-- Table Body -->
+                    <tbody v-if="item.givingNoticeParty">
+                      <tr>
+                        <td class="pl-0">
+                          <div class="mr-2">
+                            <v-icon class="notice-party-icon colour-dk-text mt-n2">
+                              {{ getNoticePartyIcon(item.givingNoticeParty) }}
+                            </v-icon>
+                          </div>
+                          <span class="notice-party-name generic-label fs-14">
+                            {{ getNoticePartyName(item.givingNoticeParty) }}
+                          </span>
+                        </td>
+                        <td>
+                          <BaseAddress
+                            :value="item.givingNoticeParty.address"
+                          />
+                        </td>
+                        <td>{{ item.givingNoticeParty.emailAddress }}</td>
+                        <td>{{ item.givingNoticeParty.phoneNumber }}</td>
+                      </tr>
+                    </tbody>
+                  </template>
+                </v-simple-table>
+              </v-col>
+            </v-row>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+      </v-expansion-panels>
+      <v-col v-else class="empty-notes-msg text-center pt-8 pb-3">
+        <p class="gray7 fs-14">A unit note has not been filed for this manufactured home.</p>
+      </v-col>
+    </v-row>
+
+  </v-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, toRefs } from 'vue-demi'
+import { MhApiStatusTypes, MhUIStatusTypes, RouteNames, UnitNoteDocTypes } from '@/enums'
+import { useRouter } from 'vue2-helpers/vue-router'
+import { personGivingNoticeTableHeaders, UnitNoteTypes } from '@/resources'
+import { UnitNoteIF } from '@/interfaces/unit-note-interfaces/unit-note-interface'
+import { pacificDate } from '@/utils'
+import { PartyIF } from '@/interfaces'
+import { BaseAddress } from '@/composables/address'
+
+export default defineComponent({
+  name: 'UnitNotePanels',
+  components: {
+    BaseAddress
+  },
+  props: {
+    unitNotes: {
+      type: Array as () => Array<UnitNoteIF>,
+      default: () => []
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup () {
+    const router = useRouter()
+
+    const localState = reactive({
+      activePanels: []
+    })
+
+    const getUnText = (noteType: UnitNoteDocTypes): string => {
```

Comment:
Should this be getUnitNoteText?

Reply:
I dunno, should it? 😆 
Can update if you like!

Reply:
Updated 👍

Reply:
Sure! Just to keep with being overly verbose as we usually do

Reply:
haha can't argue with that.